functions/src/functions/invitation.ts

```diff
@@ -72,8 +72,8 @@ export const createInvitation = validatedOnCall(
           ...request.data,
           code: invitationCode,
         }
-        await userService.createInvitation(invitation)
-        return { code: invitationCode }
+        const { id } = await userService.createInvitation(invitation)
+        return { id }
```

Comment:
Could we maybe return both id and code here? since for the user, the code would actually be more interesting... or is this just intended for the dashboard and it would get the invitation with the next request anyways with the code inside?

Reply:
Yes, this is just for identification purposes on the dashboard.

The flow is:

1. Create invitation
2. Get the id
3. Redirect to edit view, that gets full invitation anyway

Reply:
sounds good then 👍